src/components/v5/shared/NotificationBanner/NotificationBanner.tsx

```diff
@@ -40,7 +40,7 @@ const NotificationBanner: FC<NotificationBannerProps> = ({
           })}
         />
       ) : null}
-      <div className="flex flex-1 flex-col items-start gap-2 @[600px]/notificationBanner:flex-row @[600px]/notificationBanner:items-center">
+      <div className="flex flex-1 flex-col items-start gap-2 @[600px]/notificationBanner:items-center">
```

Comment:
This is just a humble suggestion @Nortsova -- to save you the hassle since this seemingly affects other areas where we use this NotificationBanner component, maybe you can add a prop for this component and adjust these styles specifically for the GroupedTransactionContent, which appears to be the only area that's being affected

And because we know your style changes are definitely fixing the issue, we can try:

```js
<div
  className={clsx(
    'flex flex-1 flex-col items-start gap-2 @[600px]/notificationBanner:flex-row @[600px]/notificationBanner:items-center',
    contentClassName,
  )}
>
```

Then in GroupedTransactionContent:

```js
<NotificationBanner
  ...existingProps
  contentClassName="@[600px]/notificationBanner:flex-col" // <<-- Add me 😄 
>
```

Reply:
Thank you @rumzledz , I decided to go with this approach. 🙌 

After some investigation, I realized that this issue arises because:
1. There's a rule for a container width of 600px (only for 720-768px does the container width become larger).
2. In UserHub, we have a `callToAction` prop containing two buttons that attempt to fill the full width.

An alternative solution I considered is to remove the `callToAction` from the component entirely.
And instead of this:
```
<NotificationBanner
 ...existingProps
  callToAction={<Buttons />}
>
```

Do something like:
```
<NotificationBanner
 ...existingProps
>
   {....content}
   <Buttons>
</NotificationBanner>
```

but this case requires me to fix a lot of styles.